nolus/assetlist.json

```diff
@@ -639,6 +639,138 @@
           }
         }
       ]
+    },
+    {
+      "description": "CRO transferred from the Osmosis mainnet that is supported on the Nolus chain",
+      "denom_units": [
+        {
+          "denom": "ibc/E1BCC0F7B932E654B1A930F72B76C0678D55095387E2A4D8F00E941A8F82EE48",
+          "exponent": 0,
+          "aliases": [
+            "basecro"
+          ]
+        },
+        {
+          "denom": "cro",
+          "exponent": 8,
+          "aliases": []
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/E1BCC0F7B932E654B1A930F72B76C0678D55095387E2A4D8F00E941A8F82EE48",
+      "name": "CRO",
+      "display": "cro",
+      "symbol": "CRO",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "ibc/E6931F78057F7CC5DA0FD6CEF82FF39373A6E0452BF1FD76910B93292CF356C1",
```

Comment:
Otherwise looks correct, but we'll need to register CRO to Osmosis' assetlist as well